### PR TITLE
Create administration.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Renamed migration guide LHS as it was badly named.

### CODE REVIEW

File: source/guides/administration.rst [hunk 1]
```diff
@@ -66,7 +66,7 @@ Workspace Management
     Statistics </manage/cloud-reporting>
     User Satisfaction Surveys </manage/cloud-user-satisfaction-surveys>
     Managing Team and Channel Members </manage/cloud-team-and-channel>
-    Cloud Data Export </manage/cloud-data-export>
+    Workspace Migration </manage/cloud-data-export>
 
 * :doc:`Mattermost Cloud Billing </manage/cloud-billing>` - Set up and manage billing for your Mattermost Cloud workspace.
```
Comment (cwarnermm) [inline source/guides/administration.rst:71]: I think you'll also need a link to the updated page in this section too: * :doc:`Workspace Migration </manage/cloud-data-export>` - Learn how to ....

**Review by cwarnermm — COMMENTED** [review]
I'm good with the change; however, it looks like this PR may introduce file duplication?

**Comment by justinegeffen** [comment]
> I'm good with the change; however, it looks like this PR may introduce file duplication?
 
Fixed it - was just a conflict with the recently-introduced data residency content that still had the previous page name in it. :)

**Review by cwarnermm — APPROVED** [review]
Thanks!

**Pull request merged**